controller/work_item_link_type.go

```diff
@@ -222,6 +223,26 @@ func (c *WorkItemLinkTypeController) List(ctx *app.ListWorkItemLinkTypeContext)
 	})
 }
 
+// ListTypeCombinations runs the list-type-combinations action.
+func (c *WorkItemLinkTypeController) ListTypeCombinations(ctx *app.ListTypeCombinationsWorkItemLinkTypeContext) error {
+	return application.Transactional(c.db, func(appl application.Application) error {
+		m, err := appl.WorkItemLinkTypeCombinations().List(ctx, ctx.WiltID)
```

Comment:
This heavily depends on the way we intend to share work item types as suggested/requested by @maxandersen . I try to come up with a document that shows the final vision for this data model.